rl/api/endpoints.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+from flask import request
+
+from rl.api import logger, app, algorithm_manager
+from logger.Logger import LogType
+
+
+@app.route("/logs")
+def get_logs():
+    """
+    Endpoint return logs, registered in RL module. It will be possible
+    to filter thelogs with query params.
+    """
+    logs = logger.get_messages()
+    some_filter = request.args.get("filter")

```

Comment:
To powinno być w odwrotnej kolejności czyli wpierw zdobywamy filtr, a potem zdobywamy wiadomości bazując na filtrze

Reply:
tu nie ma jeszcze implementacji to tylko pokazowe